docs/content/test/TestMinimized.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Test minimized build
+type: docs
+---
+
+# Testing minimized build
+
+About minimization: [link]({{< ref "/assemble/Minimization.md" >}})
+
+We want to run our ui tests against build as close as possible to production one.\
+It's quite a challenging task, mostly because of [tooling problems](https://issuetracker.google.com/issues/126429384)\
+Developers should maintain list of keep rules of code referenced from test app manually.
+
+We use [keeper](https://slackhq.github.io/keeper/) to workaround these problems.
```

Comment:
"You can **work around** this problem using a **workaround**."

https://www.macmillandictionary.com/dictionary/british/work-around